java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -43,6 +43,22 @@ public interface StreamBaseCommands {
      */
     CompletableFuture<String> xadd(String key, Map<String, String> values);
 
+    /**
+     * Adds an entry to the specified stream stored at <code>key</code>.<br>
```

Comment:
I think you need to tell what is the benefit of the new API there. Please also add links from existing APIs to new ones like: `To add entry with duplicated keys, use {@link ....`